src/main/java/com/amazon/opendistroforelasticsearch/security/resolver/IndexResolverReplacer.java

```diff
@@ -161,7 +159,7 @@ private static final boolean isLocalAll(final String... requestedPatterns) {
         return false;
     }
 
-    private Resolved resolveIndexPatterns(final IndicesOptions indicesOptions, final Object request, final String... requestedPatterns0) {
+    private Resolved resolveIndexPatterns(final IndicesOptions indicesOptions, final boolean isSearchOrFieldsCapabilities, final String... requestedPatterns0) {
```

Comment:
The only comment that is related to new variable name. Not very intuitive.
I was thinking initially to renamed it to "isSearchOrFieldsCapabilitiesRequest", but Vlad suggested a better one "enableCrossClusterResolution", since this variable is basically indicating if we need to do cross-cluster resolution based on type of the request.

Reply:
Done, sounds  better for sure